docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-api-test-tool:
+
+Ledger API Test Tool
```

Comment:
Just my opinion. You refer to it in lowercase later.

```suggestion
Ledger API test tool
```

Reply:
I don't think things have to be capitalised just because they're names, but people seem to like it - they think it makes it more of a Thing somehow. It's not really necessary, but totally up to you.

Reply:
I'd prefer keeping the approach of capitalizing the names